authenticator/server/authenticator.go

```diff
@@ -145,7 +156,6 @@ func (a *authenticator) GetPGMD5Hash(ctx context.Context, req *pb.PGMD5HashReque
 
 func (a *authenticator) GetPGSHA256Hash(ctx context.Context, req *pb.PGSHA256HashRequest) (*pb.PGSHA256Response, error) {
 	a.incrementRequestCount()
-	reqLogger := getRequestLogger(ctx)
```

Comment:
Would it be possible to move the request loggers back to the very beginning and pass them into `GetPassword`? The reason I ask is, they're super hard to discover inside the `ctx` unless you already know they're there. But, there was no other way to pass them into these methods because they have to follow an interface. By having them at the very top, it makes them much more discoverable to others and they won't either believe there's no logger available, or start using the static one again.

Reply:
Good suggestion! Implemented.